src/shared-stores/profile-store.tsx

```diff
@@ -51,7 +51,7 @@ export const useProfileStore = create<ProfileStore>()((set, get) => ({
 	refresh: async () => {
 		const promises = [
 			get().refreshUsername(),
-			get().refreshAdoptedTrees(),
+			get().refreshAdoptedTreesInfo(),
```

Comment:
@aeschi @Jaszkowic calling this function seemed more meaningful to me, instead of calling only `refreshAdoptedTrees()` which only updates an array with a tree IDs. The function `refreshAdoptedTreesInfo()` calls `refreshAdoptedTrees()` inside it, then updates the infos of each of those trees. does it make sense? or am missing/breaking something?

Reply:
makes sense to me